app/src/pages/playground/playgroundUtils.ts

```diff
@@ -230,7 +230,26 @@ export function getTemplateMessagesFromAttributes({
       messages: null,
     };
   }
-
+  if (provider === "ANTHROPIC") {
+    const { success, data } =
+      modelConfigWithInvocationParametersSchema.safeParse(parsedAttributes);
+    if (success) {
+      const messages = inputMessages.data.llm.input_messages;
+      const systemPrompt = data.llm.invocation_parameters["system"];
+      if (
+        typeof systemPrompt === "string" &&
+        systemPrompt &&
+        (!messages || messages[0].message.role !== "system")
+      ) {
```

Comment:
just for my understanding, the thought on the final check here is just that if we already have a system role message here it's probably the right one and this information was duplicated in instrumentation or something?

Reply:
That's right. Currently this check is redundant and doesn't run because the playground spans don't put the "system' parameter back into the invocation parameters--it just leaves the system message as is in the list of messages. The only time where this check is needed is if in the future playground spans start putting the "system" parameter back into the invocation parameters.